packages/fast-components/src/flipper/flipper.styles.ts

```diff
@@ -0,0 +1,99 @@
+import { css } from "@microsoft/fast-element";
+import { display, disabledCursor, focusVisible } from "../styles";
+import { SystemColors } from "../styles/system-colors";
+
+export const FlipperStyles = css`
+    ${display("inline-flex")} :host {
+        width: calc(var(--height-number) * 1px);
+        height: calc(var(--height-number) * 1px);
+        justify-content: center;
+        align-items: center;
+        margin: 0;
+        position: relative;
+        fill: var(--neutral-foreground-rest);
+        color: var--neutral-foreground-rest);
```

Comment:
looks like you forgot this "("

Reply:
Nice catch - I'll update this once the spec is approved 👍